### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  id: WSTrustSerializationException
  children:
  - System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor
  - System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.String)
  - System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: WSTrustSerializationException
  nameWithType: WSTrustSerializationException
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  type: Class
  summary: "Исключение, которое выдается при возникновении ошибки во время сериализации или десериализации сообщения WS-Trust."
  syntax:
    content: 'public class WSTrustSerializationException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  langs:
  - csharp
  name: WSTrustSerializationException()
  nameWithType: WSTrustSerializationException.WSTrustSerializationException()
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel.Protocols.WSTrust
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException&quot;> </xref>."
  remarks: "Этот конструктор инициализирует <xref:System.Exception.Message%2A>свойств нового экземпляра системным сообщением, содержащим описание ошибки.</xref:System.Exception.Message%2A> Это сообщение учитывает текущую культуру системы.       В следующей таблице показаны исходные значения свойств для экземпляра.      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Строка локализованного сообщения об. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WSTrustSerializationException ();
    parameters: []
  overload: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  langs:
  - csharp
  name: WSTrustSerializationException(String)
  nameWithType: WSTrustSerializationException.WSTrustSerializationException(String)
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel.Protocols.WSTrust
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException&quot;> </xref> класса с указанным сообщением об ошибке."
  remarks: "Этот конструктор инициализирует <xref:System.Exception.Message%2A>нового исключения, используя свойство `message` параметр.</xref:System.Exception.Message%2A> Содержимое `message` предназначен должно быть понятно пользователю. Код, вызывающий этот конструктор, должен убедиться, что эта строка локализована для текущего языка и региональных параметров системы.       В следующей таблице показаны исходные значения свойств для экземпляра.      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WSTrustSerializationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение об ошибке с объяснением причин исключения."
  overload: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  langs:
  - csharp
  name: WSTrustSerializationException(SerializationInfo,StreamingContext)
  nameWithType: WSTrustSerializationException.WSTrustSerializationException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel.Protocols.WSTrust
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException&quot;> </xref> сериализованными данными."
  remarks: "Данный конструктор вызывается в процессе десериализации для воссоздания объекта исключения, переданного в потоке."
  syntax:
    content: protected WSTrustSerializationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Объект <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> объект, содержащий сериализованные данные объекта."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Объект <xref:System.Runtime.Serialization.StreamingContext>, содержащий контекстные сведения об источнике или назначении.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  langs:
  - csharp
  name: WSTrustSerializationException(String,Exception)
  nameWithType: WSTrustSerializationException.WSTrustSerializationException(String,Exception)
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel.Protocols.WSTrust
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException&quot;> </xref> класса с указанным сообщением об ошибке и ссылкой на внутреннее исключение, вызвавшее данное исключение."
  remarks: "Исключение, созданное как прямой результат предыдущего исключения, должно содержать ссылку на предыдущее исключение в <xref:System.Exception.InnerException%2A>свойство.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Свойство возвращает то же значение, которое было передано в конструктор.</xref:System.Exception.InnerException%2A>       В следующей таблице показаны исходные значения свойств для экземпляра.      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Ссылка на внутреннее исключение. |   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WSTrustSerializationException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение об ошибке с объяснением причин исключения."
    - id: inner
      type: System.Exception
      description: "<xref:System.Exception>, Являющееся причиной текущего исключения.</xref:System.Exception> Если `inner` не <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, текущее исключение сгенерировано в <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> блок, обрабатывающем внутреннее исключение."
  overload: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  isExternal: false
  name: WSTrustSerializationException()
  nameWithType: WSTrustSerializationException.WSTrustSerializationException()
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException()
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.String)
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  isExternal: false
  name: WSTrustSerializationException(String)
  nameWithType: WSTrustSerializationException.WSTrustSerializationException(String)
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  isExternal: false
  name: WSTrustSerializationException(SerializationInfo,StreamingContext)
  nameWithType: WSTrustSerializationException.WSTrustSerializationException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  isExternal: false
  name: WSTrustSerializationException(String,Exception)
  nameWithType: WSTrustSerializationException.WSTrustSerializationException(String,Exception)
  fullName: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.WSTrustSerializationException(String,Exception)
- uid: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException.#ctor*
  parent: System.IdentityModel.Protocols.WSTrust.WSTrustSerializationException
  isExternal: false
  name: WSTrustSerializationException
  nameWithType: WSTrustSerializationException.WSTrustSerializationException
